typhos/cache.py

```diff
@@ -210,3 +225,112 @@ def get(self, obj):
             if desc is not None:
                 # In certain scenarios (such as testing) this might happen
                 self._new_description(obj, desc)
+
+
+# The default stale cached_path threshold time, in seconds:
+TYPHOS_PATH_CACHE_TIME = int(os.environ.get('TYPHOS_PATH_CACHE_TIME', '600'))
```

Comment:
I like the idea of having the default value and the opt-out aspect of it since this addition brings much value.